mailpoet/assets/css/src/components-plugin/_listing.scss

```diff
@@ -160,18 +157,16 @@ div.mailpoet-listing-bulk-actions-container {
 
   thead th,
   thead th a {
-    color: $color-text-light !important;
+    color: #2c3338 !important;
```

Comment:
Can we have a descriptive color variable instead?

Reply:
do you have a suggestion? @samnajian

Reply:
No specific opinion, maybe something like `$color-text-dark` ? these colors are defined in `mailpoet/assets/css/src/settings/_colors.scss`, but since we are just starting the styling touches it makes sense to have the color assigned to a variable so that we can re-use it and not copy paste or pick other colors by mistake.

Reply:
That follows the precedence of the naming set in _colors.scss, so that makes sense to me. Thanks! Addressed in [81db09c](https://github.com/mailpoet/mailpoet/pull/4200/commits/81db09c5c4346bb26928f98746cbc825c28f72c2)